fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -352,7 +352,7 @@ public Response createOrReplaceObjectRdf(
         final RdfStream resourceTriples;
 
         if (resource.isNew()) {
-            resourceTriples = new RdfStream();
+            resourceTriples = new DefaultRdfStream(translator().reverse().convert(resource()).asNode());
```

Comment:
We do not necessarily need to address it here, but we have `translator().reverse().convert(resource())` all over the place. Would it make sense to have a helper method on `IdentifierConverter` to simplify this?